addons/board/static/src/board_controller.scss

```diff
@@ -27,7 +27,6 @@
 }
 
 .o-dashboard-action {
-    background-color: white;
     align-self: start;
     border: 1px solid;
     border-color: #e5e5e5 #dbdbdb #d2d2d2;
```

Comment:
Here, the color of the border is also hard coded and it appears too bright in dark mode. You can just remove `border` and `border-color` properties and add `border` class to `o-dashboard-action`